src/solver/simulation/timeseries-numbers.cpp

```diff
@@ -639,6 +678,18 @@ void drawAndStoreTSnumbersForNOTintraModal(const array<bool, timeSeriesCount>& i
               = (uint32)(floor(study.runtime->random[seedTimeseriesNumbers].next() * nbTimeSeries));
         }
 
+        // -------------
+        // Hydro Power Credits...
+        // -------------
+        indexTS = ts_to_tsIndex.at(timeSeriesHydroPowerCredits);
+
+        if (!isTSintramodal[indexTS])
+        {
+            uint nbTimeSeries = area.hydro.series->maxgen.width;
+            area.hydro.series->timeseriesNumbersPowerCredits[0][year]
+              = (uint32)(floor(study.runtime->random[seedTimeseriesNumbers].next() * nbTimeSeries));
```

Comment:
```suggestion
            uint nbTimeSeries = area.hydro.series->maxgen.width;
            if (nbTimeSeries != 1) {
            area.hydro.series->timeseriesNumbersPowerCredits[0][year]
              = (uint32)(floor(study.runtime->random[seedTimeseriesNumbers].next() * nbTimeSeries));
              }
```
By default, the number of columns is equal to 1. So this draw won't be made and results won't change.

Results will only change if the user adds more time-series.

Reply:
Work fine. Thanks!